src/js/components/Notification/__tests__/Notification-test.js

```diff
@@ -155,4 +159,78 @@ describe('Notification', () => {
     });
     expect(onClose).not.toHaveBeenCalled();
   });
+
+  test('custom theme', () => {
+    const onOpen = jest.fn();
+
+    const theme = {
+      notification: {
+        direction: 'row',
+        truncation: true,
+        container: {
+          pad: 'medium',
+        },
+        toast: {
+          direction: 'column',
+          truncation: false,
+        },
+        critical: {
+          background: 'red',
+          toast: {
+            background: 'background-front',
+          },
+        },
+      },
+    };
+
+    const Test = () => {
+      const [visible, setVisible] = useState(false);
+      return (
+        <Grommet theme={theme}>
+          <TestNotification status="critical" />
+          <Button
```

Comment:
Do we need the button interaction for testing toast notifications? Can we just render the Notification?

Reply:
True, we don't need it. I have simplified the story.